server/lib/notifications.js

```diff
@@ -137,6 +141,61 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+async function notifyAdminsOfCollectiveAndCheckUserExpensesAmount(activity) {
+  try {
+    await notifyAdminsOfCollective(activity.data.collective.id, activity);
+    await notifyUserIfExpensesExceededMinimumFormAmount(activity);  
+  } catch (error) {
+    console.log(error);
+  }
+  
+}
+      
+async function notifyUserIfExpensesExceededMinimumFormAmount(activity) {
+  try {
+    const host = await models.Collective.findById(activity.data.host.id);
+    if (host.currency && host.currency !== 'USD') {
+      return;
+    }
+    // ideally change to consider the current fiscal year instead(Example: might be from
+    // march to february instead of from the beginning from january)
+    const beginningOfCurrentYear = new Date(new Date().getFullYear(), 0, 1);
+    const userExpenses = await models.Expense.findAll({
+      where: {
```

Comment:
better to also specify the `attributes` array to only fetch what you really need (in this case, it looks like you only need `['currency', 'amount', 'status']`